src/Tickets/Emails/Email_Abstract.php

```diff
@@ -468,7 +470,43 @@ public function get_default_additional_content(): string {
 	 * @return WP_Post|null;
 	 */
 	public function get_post() {
-		return get_page_by_path( $this->id, OBJECT, Email_Handler::POSTTYPE );
+		return get_page_by_path( $this->id, OBJECT, Post_Type::SLUG );
+	}
+
+	/**
+	 * Creates the Post in the Database for this Email.
+	 *
+	 * @since TBD
+	 *
+     * @return int|WP_Error The post ID on success. The value 0 or WP_Error on failure.
```

Comment:
Indent instead of spaces.
```suggestion
	 * @return int|WP_Error The post ID on success. The value 0 or WP_Error on failure.
```